Dockerfile

```diff
@@ -10,13 +10,12 @@ EXPOSE 80
 
 RUN apt-get update -y && \
     apt-get -y dist-upgrade && \
-    apt-get install -y build-essential \
+    apt-get install -y libz-dev
```

Comment:
```suggestion
    apt-get install -y libz-dev \
                        build-essential                                    
```

Reply:
@aloncarmel111 Is it really worth additional 30 minutes of changing-pushing-waiting for build etc?

Reply:
actually build-essential is already installed in base-image so we can removed this

Reply:
Is it a necessary part of base image?
Base should be as small as possible.
My suggestion is to remove it from the base.

Reply:
ok let's create followup task to removed build-essential from base-image